source/game_sa/GxtChar.cpp

```
@@ -13,8 +13,10 @@ void AsciiToGxtChar(const char* src, GxtChar* dst) {
 }
 
 // NOTSA -- Derived from GxtCharToAscii (0x69F7E0)
-// FIX_BUGS (sorta): Make caller provide size instead of creating a static buffer.
 char* GxtCharToUTF8(char* out, const GxtChar* src, size_t size, size_t offset) {
+    if (!out || !src || size == 0 || offset >= size)
+        return out;
```

No silent errors please

---

?